docs/src/examples.md

```diff
@@ -0,0 +1,139 @@
+Examples
+========
+
+The K8sClusterManager is intended to be used inside a pod running on a Kubernetes cluster.
```

Comment:
```suggestion
The K8sClusterManager is intended to be used inside a `Pod` running on a Kubernetes cluster.
```

I'd use `Capital` when referring to K8s resources

Reply:
Okay. As I tend to use backticks for code I think I'll italicize that instead. Additionally, seems reasonable to make the first reference to _Pod_ to be a link

Reply:
For the plural form of pod I've gone with _Pod_s instead of _Pods_ even though it makes my text editor unhappy

Reply:
Oh, interesting. GitHub doesn't handle that but Documenter.jl does

Reply:
One more comment on this: as `kubectl` allows `get $resource` where resource is case-insensitive and either plural or singular I'm not I should be capitalizing the K8s resources

Reply:
> I'm not I should be capitalizing the K8s resources

I follow the K8s doc convention on this, which is to capitalize them e.g. https://kubernetes.io/docs/concepts/workloads/pods/

Reply:
Thanks for the reference. I'll follow that convention and only use the capitalization as I think that works well